docker/metrics-collector/config.grafana.env

```diff
@@ -0,0 +1,2 @@
+GF_SECURITY_ADMIN_PASSWORD=foobar
```

Comment:
`<paranoid>`Just to double-check: is this supposed to be committed?`</paranoid>`

Reply:
It's fine. Grafana will run locally and just required to access it. I could go with defaults, though to get rid of this file.